src/Files.App/Utils/Global/QuickAccessManager.cs

```diff
@@ -42,11 +42,10 @@ private void PinnedItemsWatcher_Changed(object sender, FileSystemEventArgs e)
 		public async Task InitializeAsync()
 		{
 			PinnedItemsModified += Model.LoadAsync;
+			await Model.LoadAsync();
```

Comment:
Is there a reason this was reorderd?

Reply:
In the line below we check with Model.PinnedFolers.Contains.... if the Recycle Bin is already pinned. LoadAsync needs to be called first, otherwise the PinnedFolders list is empty and the check would not work.

Reply:
Makes sense.